xline/src/server/kv_server.rs

```diff
@@ -391,13 +386,13 @@ where
     S: StorageApi,
 {
     /// Range gets the keys in the range from the key-value store.
-    #[instrument(skip(self))]
+    #[instrument(skip_all)]
```

Comment:
Same as above.
```suggestion
    #[instrument(skip(self), field(range_request = request.get_ref()))]
```

Reply:
Instrument is not suitable for putting longer data, because each `event` in `span` will print these contents